Library/ViewModels/ThanksViewModel.swift

```diff
@@ -6,7 +6,8 @@ import ReactiveSwift
 public typealias ThanksPageData = (
   project: Project,
   reward: Reward,
-  checkoutData: KSRAnalytics.CheckoutPropertiesData?
+  checkoutData: KSRAnalytics.CheckoutPropertiesData?,
+  context: KSRAnalytics.TypeContext.PledgeContext
```

Comment:
Related to a few of the comments above but I think we can delete this parameter because the value will always be the same. In the method for the event we can provide the context.